src/IdeaForm.js

```diff
@@ -25,13 +30,13 @@ const IdeaForm = ({ onSubmit, idea }) => {
       button: "Add",
     },
     editForm: {
-      header: "Would you like to change your Idea?",
-      label: "New Idea:",
+      header: "Enter Idea Change",
+      label: "Change Idea:",
       button: "Change",
     },
   };
 
-  const formType = ideaState.id ? "editForm" : "addForm";
+  const formType = ideaState?.id ? "editForm" : "addForm";
```

Comment:
When will `ideaState` be undefined?

Reply:
It will not!
I think this was part of what was lost when I messed things up.